gst/meson.build

```diff
@@ -8,5 +8,7 @@ endif
 subdir('nnstreamer')
 
 if get_option('enable-datarepo')
+  json_glib_dep = dependency('json-glib-1.0', required : true, \
```

Comment:
You declared `json_glib_dep` here, so please remove it from /gst/datarepo/meson.build.

Reply:
I got it. I removed json_glib_dep in /gst/datarepo/meson.build
Thanks for your comment!